scripts/cluster-scanner/internal/search_clusters.go

```diff
@@ -0,0 +1,16 @@
+package internal
+
+import (
+	"log/slog"
+
+	"github.com/spectrocloud/palette-sdk-go/api/models"
+	"github.com/spectrocloud/palette-sdk-go/client"
+)
+
+func SearchClusters (paletteClient *client.V1Client, logger *slog.Logger) ([]*models.V1SpectroClusterSummary, error) {
+	// Search for clusters
+	logger.Info("Searching for clusters...")
```

Comment:
If you move this print to main before you invoke the function on line 41, then you don't need the logger as a function dependency here, which would be much cleaner.